auth/src/main/java/org/aerogear/mobile/security/checks/DeveloperModeCheck.java

```diff
@@ -19,8 +19,9 @@
     /**
      * Check if developer mode has been enabled in the device.
      *
-     * @param context Context to be used by the check.
-     * @return <code>true</code> if the developer options have been enabled on the device.
+     * @param context {@link Context} to be used by the check
+     * @return <code>true</code> if the developer options have been enabled on the device
+     * @throws IllegalArgumentException if context is null
```

Comment:
IllegalArgumentException is not thrown here. Context is guaranteed to be non null by the parent class.